.github/workflows/agentics-maintenance.yml

```
@@ -0,0 +1,582 @@
+on:
+  schedule:
+    - cron: "37 */2 * * *"  # Every 2 hours (based on minimum expires: 1 days)
+  workflow_dispatch:
+    inputs:
+      operation:
+        description: 'Optional maintenance operation to run'
+        required: false
+        type: choice
+        default: ''
+        options:
+          - ''
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)